drv/lpc55-sprot-server/src/main.rs

```diff
@@ -295,7 +292,18 @@ impl Io {
                 // potentially throttle sends in the future.
                 self.spi.rxerr_clear();
                 self.stats.rx_overrun = self.stats.rx_overrun.wrapping_add(1);
-                err = Some(IoError::Flow);
+                // If we were just sending our response, and SP was
+                // just sending zeros and we received the first byte
+                // correctly and that first byte was zero, then
```

Comment:
Where do we check that the first byte was zero?

Reply:
I fixed up the comment here to refer to `Protocol::Ignore` instead of `byte was zero`.